Gems/Atom/RPI/Code/Tests/Material/MaterialTypeSourceDataTests.cpp

```diff
@@ -1979,60 +2010,70 @@ namespace UnitTest
         CheckPropertyValue<Data::Asset<ImageAsset>>(materialTypeAsset, Name{ "general.relative" }, m_testImageAsset2);
     }
 
-
-    TEST_F(MaterialTypeSourceDataTests, CreateMaterialTypeAsset_Error_UnsupportedVersionUpdate)
-    {
-        MaterialTypeSourceData sourceData;
-
-        MaterialTypeSourceData::PropertyDefinition* propertySource = sourceData.AddPropertyGroup("general")->AddProperty("a");
-        propertySource->m_dataType = MaterialPropertyDataType::Int;
-        propertySource->m_value = 0;
-
-        sourceData.m_version = 2;
-
-        MaterialTypeSourceData::VersionUpdateDefinition versionUpdate;
-        versionUpdate.m_toVersion = 2;
-        MaterialTypeSourceData::VersionUpdatesRenameOperationDefinition updateAction;
-        updateAction.m_operation = "operationNotKnown";
-        versionUpdate.m_actions.push_back(updateAction);
-        sourceData.m_versionUpdates.push_back(versionUpdate);
-
-        ErrorMessageFinder errorMessageFinder;
-        errorMessageFinder.AddExpectedErrorMessage("Unsupported material version update operation 'operationNotKnown'");
-        errorMessageFinder.AddIgnoredErrorMessage("Failed to build MaterialTypeAsset", true);
-
-        auto materialTypeOutcome = sourceData.CreateMaterialTypeAsset(Uuid::CreateRandom());
-        EXPECT_FALSE(materialTypeOutcome.IsSuccess());
-
-        errorMessageFinder.CheckExpectedErrorsFound();
-    }
-    
-    TEST_F(MaterialTypeSourceDataTests, CreateMaterialTypeAsset_Error_VersionInWrongLocation)
+    TEST_F(MaterialTypeSourceDataTests, CreateMaterialTypeAsset_ResolveSetValueVersionUpdates)
```

Comment:
Did you delete CreateMaterialTypeAsset_Error_UnsupportedVersionUpdate on purpose? Maybe it was moved elsewhere?

Reply:
The removal of MaterialTypeSourceDataTests.CreateMaterialTypeAsset_Error_UnsupportedVersionUpdate was intentional as all error-checking is now concentrated in MaterialVersionUpdate.cpp instead of in the MaterialSourceData files. 
The equivalent test is now MaterialTypeAssetTests.Error_InvalidMaterialVersionUpdate_UnknownOperation. (I probably should have mentioned that explicitly in the commit message as the apparent removal of a test case can indeed seem rather suspect!)